test/higher_order_ops/test_with_effects.py

```diff
@@ -0,0 +1,191 @@
+# Owner(s): ["module: functorch"]
+import unittest
+
+import torch
+import torch._dynamo
+import torch._inductor
+import torch._inductor.decomposition
+from torch._functorch.aot_autograd import aot_export_module
+from torch._higher_order_ops.effects import with_effects
+from torch._higher_order_ops.torchbind import enable_torchbind_tracing
+from torch.fx.experimental.proxy_tensor import make_fx
+from torch.testing import FileCheck
+from torch.testing._internal.common_utils import (
+    find_library_location,
+    IS_FBCODE,
+    IS_MACOS,
+    IS_SANDCASTLE,
+    IS_WINDOWS,
+    run_tests,
+    TestCase,
+)
+
+
+@unittest.skipIf(not torch._dynamo.is_dynamo_supported(), "dynamo isn't support")
+class TestWithEffects(TestCase):
+    def setUp(self):
+        if IS_MACOS:
+            raise unittest.SkipTest("non-portable load_library call used in test")
+        elif IS_SANDCASTLE or IS_FBCODE:
+            torch.ops.load_library(
+                "//caffe2/test/cpp/jit:test_custom_class_registrations"
+            )
+        elif IS_WINDOWS:
+            lib_file_path = find_library_location("torchbind_test.dll")
+            torch.ops.load_library(str(lib_file_path))
+        else:
+            lib_file_path = find_library_location("libtorchbind_test.so")
+            torch.ops.load_library(str(lib_file_path))
```

Comment:
nit: Could we do something like mark the below tests with skipIf decorators if they use torchbind and torchbind wasn't found?

I'm not sure why, but I'm unable to actually find libtorchbind_test.so

Reply:
I ended up skipping the test the entire test directly because I think I need to enable tokens on export before allowing torchbind ops 😅